### PR TITLE
Lesson_3_1

### PR DESCRIPTION
**Pull request by betymesfin**

### CODE REVIEW

File: src/index.css
Comment (codiini) [inline src/index.css:1]: Convert this to a modular stylesheet and use it for styling.

**Review by codiini — CHANGES_REQUESTED** [review]

**Comment by codiini** [comment]
Great job with the custom font and background image, but convert the index.css file to use modules and you're all good.

File: src/index.css
Comment (betymesfin) [reply src/index.css:1]: done

**Comment by codiini** [comment]
> Great job with the custom font and background image, but convert the index.css file to use modules and you're all good.

Hey Beth, sorry about the back and forth, but please change it to a regular CSS stylesheet. I sent you a message on Slack

**Pull request merged**